CONTRIBUTING.md

```diff
@@ -31,6 +31,26 @@ pnpm install
 
 5. **Install Docker**: Install Docker using [Docker desktop](https://www.docker.com/products/docker-desktop) or [Colima](https://github.com/abiosoft/colima)
 
+6. **Run Docker Compose**: Depending on your system, use the appropriate Docker Compose file to set up your environment.
+
+- On Intel Macs:
+
+```sh 
+docker compose -f docker-compose-mac.yml up
+```
+
+- On Apple Silicon Macs:
+
+```sh 
+docker compose -f docker-compose-apple-silicon.yml up
+```
+
+- On Windows:
+
+```sh 
+docker-compose up --build
```

Comment:
Is `--build` required?

Reply:
No but I need to test on windows to be sure